tests/EventTest.php

```diff
@@ -97,8 +98,8 @@ public function testFiresNotIfNoSessionLanguageChange()
     public function testFiresNotIfPersistenceDisabled()
     {
         $this->mockUrlManager([
-            'languages' => ['fr', 'en', 'de'],
-            'on languageChanged' => [$this, 'languageChangedHandler'],
+            'languages'                 => ['fr', 'en', 'de'],
+            'on languageChanged'        => [$this, 'languageChangedHandler'],
```

Comment:
We don't align `=>` and I don't want to introduce this. (Had this before and it only leads to trouble e.g. when modifying code that requires a change of alignment depths: you won't easily see the actual changed line in git history).